src/components/KRadio/KRadio.vue

```diff
@@ -351,23 +344,24 @@ $kRadioDotSize: 6px;
     .radio-card-wrapper {
       align-items: center;
       background-color: var(--kui-color-background, $kui-color-background);
+      border: var(--kui-border-width-10, $kui-border-width-10) solid var(--kui-color-border, $kui-color-border);
```

Comment:
I think we need to use a box-shadow instead of border for the selected cards. If no cards are selected, and then you select one, the cards "jump" on the page in their container when the border is added. Alternatively, you could give all cards the same border-width border, but `transparent` by default, and then add the color when the item is selected.

Reply:
Fixed